Chess tournament website

Chaseknis opened the pull request:
### Add chess tournament website

- Add index.html file
- Add style.css file
- Add about.html file
- Add linters.yml file
- Add .gitignore file
- No linter errors
- Add responsive mobile and desktop versions
- Add javascript objects for the council section
- Add a hamburger menu using javascript
- Add functionality to the more button using javascript

Chaseknis committed:
add index.html file

Chaseknis committed:
add style.css file

Chaseknis committed:
Add .gitignore file
Add linters
Check for linter error

Chaseknis committed:
Add index.html file
Add node package
Add .gitignore file
Add linters
Check for linter errors
Add LICENSE file
Add mobile version
Add responsive desktop website version
Add about.html file
Add images folder

Chaseknis committed:
Update README.md file

Chaseknis committed:
Add javascript dynamically in the council section

Chaseknis committed:
Merge branch 'mobile_preview' of https://github.com/Chaseknis/first_capstone_project into mobile_preview

Chaseknis committed:
Update README.md

Chaseknis committed:
Add about page sections

Chaseknis committed:
Add the more button to the council's page

Chaseknis committed:
Add Hmaburger menu using javascript

Chaseknis committed:
Add javascript functionality on the more button

Chaseknis committed:
check for linter errors

Chaseknis committed:
Update README.md

Chaseknis committed:
Remove commented codes from the project

Chaseknis committed:
Merge branch 'mobile_preview' of https://github.com/Chaseknis/first_capstone_project into mobile_preview

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+
+    </main>   
+    
+    <footer>
+        <div class="footer_wrapper">
+            <img src="images/logo.png" alt="logo">
+            <p>
+                @2023 International Chess Masters Tournament. All Rights reserved.
+            </p>
+
+        </div>
+
+    </footer>
```
- [x] Awesome job with the design. In the design template, the footer section on the homepage has a white background while on the about page, it has a dark background. It would be great if you update it accordingly. ✍️

**Home - Desktop**
![Screenshot from 2023-01-04 19-18-23](https://user-images.githubusercontent.com/17243325/211071092-c1ffe7a7-7b58-450e-ac11-6a5593202143.png)

**About - Desktop**
![Screenshot from 2023-01-04 19-17-45](https://user-images.githubusercontent.com/17243325/211071081-d56702fc-3938-4439-a4e0-8d6c7c74f78a.png)

**Note**
- [x] In the mobile version, please use a white background for the footer on both pages. 😎️

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+                <hr>
+
+                <div class="cards_container">
+                    <div class="card_wrapper">
```
- [x] Since this element is self-contained composition it will be more suitable to use an `<article>` element instead of a `<div>`. Please check this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to get more information about the `article` semantic tag. 🥸️

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+                        </p>
+                    </div>
+
+                    <div class="card_wrapper">
```
- [x] Kindly use the semantic `article` tag instead of a `div` for semantic purposes. ✍️

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+                        </p>
+                    </div>
+
+                    <div class="card_wrapper">
```
- [x] Kindly use the semantic `article` tag instead of a `div` for semantic purposes. ✍️

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+                        </p>
+                    </div>
+
+                    <div class="card_wrapper">
```
- [x] Kindly use the semantic `article` tag instead of a `div` for semantic purposes. ✍️

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+                        </p>
+                    </div>
+
+                    <div class="card_wrapper">
```
- [x] Kindly use the semantic `article` tag instead of a `div` for semantic purposes. ✍️

amejid commented on index.html:
```diff
@@ -0,0 +1,254 @@
+            </div>
+            <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
+            <div class="menu_links_container">
+                <ul class="nav_menu">
+                    <a class="menu_link" href="about.html"><li>About</li></a>
+                    <a class="menu_link" href="#"><li>Program</li></a>
+                    <a class="menu_link" href="#"><li>Join</li></a>
+                    <a class="menu_link" href="#"><li>Sponsor</li></a>
+                    <a class="menu_link" href="#"><li>News</li></a>
+                </ul>
```
- [x] Kindly update the structure of this list in the order `ul`, `li`, and `a` as it will make more sense semantically. Please check the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) about `ul` tags to get more insight into what elements are allowed inside them. 🤓️

```
<ul>
  <li><a>About</a></li>
  ...
</ul>
```

amejid commented on main.js:
```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-vars */
```
- [x] Please remove this line so the ESLint can work properly. We are on a quest to achieve something called "Clean Code" and linters seem to help a lot on that path. 💯️

amejid commented on main.js:
```diff
@@ -0,0 +1,91 @@
+if (councilCards) {
+  for (let i = 0; i < councilList.length; i += 1) {
+    councilCards.innerHTML += `
+      <div class="judges">
```
- [x] Let us use an `article` tag instead of `div` for the reasons discussed earlier. 😎️

amejid commented on about.html:
```diff
@@ -0,0 +1,194 @@
+        <section class="about_homepage">
+
+            <div class="small_menu_wrapper">
+                <a href="index.html"><img src="images/icons8-home-page-48.png" alt=""></a>
```
- [x] Please add the `alt` attribute on this image because we want to make our website accessible to everyone. Screen Readers will thank you later. 🧐️

amejid commented on about.html:
```diff
@@ -0,0 +1,194 @@
+            </div>
+            <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
+            <div class="menu_links_container">
+                <ul class="nav_menu">
+                    <a class="menu_link" href="index.html"><li>Home</li></a>
+                    <a class="menu_link" href="#"><li>Program</li></a>
+                    <a class="menu_link" href="#"><li>Join</li></a>
+                    <a class="menu_link" href="#"><li>Sponsor</li></a>
+                    <a class="menu_link" href="#"><li>News</li></a>
+                </ul>
```
- [x] Please use the following structure for the reasons discussed earlier. 🤝️

```
<ul>
  <li><a>About</a></li>
  ...
</ul>
```

amejid commented on about.html:
```diff
@@ -0,0 +1,194 @@
+
+        <section class="about_partners">
+            <div class="about_partners_wrapper">
+                <div class="about_parteners_container">
+                    <h2>
+                        See the Past International Chess Masters Tournaments
+                    </h2>
+                    <hr>
+                    <p>
+                        Take a look at the last two Chess Masters Tournaments which took place in Venice and Munich
+                    </p>
+                </div>
```
- [x] Awesome design. Kindly fix this orange `<hr>` directly under the text, we don't want it hanging on the left side on its own. And just like that "PERFECTION". 💯️💯️💯️

![Screenshot from 2023-01-06 20-57-03](https://user-images.githubusercontent.com/17243325/211077084-f18f62a8-eeda-47f4-b882-393a80af4fc4.png)

amejid reviewed (changes requested):
Hi @Chaseknis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif"/>

## Highlights
- Professional README :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Meaningful commits :heavy_check_mark: 
- Awesome design :fire: :fire: :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Chaseknis committed:
Added the right html semantic
Add the line alignment in the about page
Adjust the changes according to the review

Kossi-stack reviewed (approved):
## Status: Approved ✔️

**Hi @Chaseknis,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

You did well by implementing all the requirements requested by the first reviewer.

![128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c](https://user-images.githubusercontent.com/81444338/150871274-7d1b779c-88e6-420b-a530-b7b905eaa3ac.gif)

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

Chaseknis commented:
@Kossi-stack Thank you!
